src/test/java/org/jabref/logic/importer/fetcher/CollectionOfComputerScienceBibliographiesFetcherTest.java

```diff
@@ -0,0 +1,116 @@
+                        "\t\t Static Architecture Conformance Checking (SACC) to\r\n" +
+                        "\t\t prevent violations to the intended architecture.\r\n" +
+                        "\t\t Measurements mined from source code repositories such\r\n" +
+                        "\t\t as code churn and code ownership has been linked to\r\n" +
+                        "\t\t several problems, for example fault proneness, security\r\n" +
+                        "\t\t vulnerabilities, code smells, and degraded\r\n" +
+                        "\t\t maintainability. The root cause of such problems can be\r\n" +
+                        "\t\t architectural. To determine the impact of the\r\n" +
+                        "\t\t refactoring of JabRef, we measure the code churn and\r\n" +
+                        "\t\t code ownership before and after the refactoring and\r\n" +
+                        "\t\t find that large files with violations had a\r\n" +
+                        "\t\t significantly higher code churn than large files\r\n" +
+                        "\t\t without violations before the refactoring. After the\r\n" +
+                        "\t\t refactoring, the files that had violations show a more\r\n" +
+                        "\t\t normal code churn. We find no such effect on code\r\n" +
+                        "\t\t ownership. We conclude that files that contain\r\n" +
+                        "\t\t violations detectable by SACC methods are connected to\r\n" +
+                        "\t\t higher than normal code churn.")
+                .withField(StandardField.TYPE, "info:eu-repo/semantics/conferenceObject")
+                .withField(new UnknownField("description"), "Information and Software Qualtiy")
+                .withField(StandardField.PAGES, "152--158")
+                .withField(new UnknownField("bibsource"), "OAI-PMH server at www.diva-portal.org")
+                .withField(new UnknownField("rights"), "info:eu-repo/semantics/openAccess")
+                .withField(StandardField.URL, "http://urn.kb.se/resolve?urn=urn:nbn:se:lnu:diva-68408")
+                .withField(new UnknownField("oai"), "oai:DiVA.org:lnu-68408")
+                .withField(StandardField.TITLE, "The relationship of code churn and architectural violations in the open source software JabRef")
+                .withField(StandardField.PUBLISHER, "Linn{\\'e}universitetet, Institutionen f{\\\"o}r datavetenskap (DV); Linn{\\'e}universitetet, Institutionen f{\\\"o}r datavetenskap (DV); Linn{\\'e}universitetet, Institutionen f{\\\"o}r datavetenskap (DV); New York, NY, USA")
+                .withField(StandardField.LANGUAGE, "eng")
+                .withField(StandardField.AUTHOR, "Tobias Olsson and Morgan Ericsson and Anna Wingkvist")
+                .withField(StandardField.YEAR, "2017");
+
+        assertTrue(searchResult.contains(firstBibEntry));
```

Comment:
Could you add a comment: "// Checking entries in the set as the query is generic and returns a changing result set"